tensorrt_llm/_torch/models/modeling_llava_next.py

```
@@ -24,7 +24,7 @@
 from .modeling_auto import AutoModelForCausalLM
 from .modeling_clip import CLIPVisionModel
 from .modeling_multimodal_utils import fuse_input_embeds
-from .modeling_utils import ModelConfig, filter_weights, register_auto_model
+from .modeling_utils import ModelConfig, filter_weights, register_auto_model, register_vision_encoder
```

Ruff (0.12.2)

Redefinition of unused `ModelConfig` from line 23

(F811)